package/yast2.changes

```diff
@@ -1,4 +1,13 @@
 -------------------------------------------------------------------
+Tue Jul 28 10:50:05 UTC 2015 - ancor@suse.com
+
+- Properly display error information when running in command line
```

Comment:
Yes, *error* is more general but my point is that the ones via Report.Error *were* reported, all at once in the end. The problem is just the exceptions, since they escaped this collector and went via the still disabled Report.Error.